source/slang/hlsl.meta.slang

```diff
@@ -172,13 +172,13 @@ vector<T,N+1> __makeVector(vector<T,N> vec, T scalar);
 
 __magic_type(TextureType)
 __intrinsic_type($(kIROp_TextureType))
-struct __TextureImpl<T, Shape: __ITextureShape, let isArray:int, let isMS:int, let sampleCount:int, let access:int, let isShadow:int, let isCombined:int, let format:int>
```

Comment:
Let's not modify this just for samplerRect. That's a legacy that can now be mapped to just sampler2D.

Reply:
I will remove isRectangle